gn3/api/streaming.py

```diff
@@ -0,0 +1,26 @@
+""" File contains endpoint for computational streaming"""
+import os
+from flask import current_app
+from flask import jsonify
+from flask import Blueprint
+from flask import request
+
+streaming = Blueprint("stream", __name__)
+
+
+@streaming.route("/<identifier>",  methods=["GET"])
+def stream(identifier):
+    """ This endpoint streams stdout from a file.
+    It expects the indetifier to be the filename
```

Comment:
*nitpick:* Fix minor typo
```suggestion
    It expects the indentifier to be the filename
```